custom_components/tuya_local/devices/water_dispenser_kg1.yaml

```diff
@@ -0,0 +1,199 @@
+name: Inline Water Dispenser
+products:
+  - id: szx3eawxigxumxi5
+    name: KG1
+    manufacturer: Casdon
+    model: Y-GX1200-KG1
+
+entities:
+  - entity: switch
```

Comment:
Consider using a valve entity, with `class: water`
No name or icon should be necessary, and default HA translations icons for water valve will be used.